engine/src/main/java/org/terasology/world/block/loader/BlockFamilyDefinitionFormat.java

```diff
@@ -309,8 +304,8 @@ private BlockFamilyDefinitionData createBaseData(JsonObject jsonObject) {
                 Optional<BlockFamilyDefinition> baseDef = assetManager.getAsset(basedOn.getAsString(), BlockFamilyDefinition.class);
                 if (baseDef.isPresent()) {
                     BlockFamilyDefinitionData data = baseDef.get().getData();
-                    if (data.getFamilyFactory() instanceof FreeformBlockFamilyFactory) {
-                        data.setFamilyFactory(null);
+                    if (data.getBlockFamily() == FreeformFamily.class) {
```

Comment:
This doesn't immediately make sense to me but maybe it would if you were debugging the code?

Reply:
Its based on changes from the old implementation. It does not make immediate sense to me. ummm